installer/scripts/service_control

```diff
@@ -69,29 +145,53 @@ verify_privileges()
     fi
 }
 
+this_omsagent_running() {
+    if [ -f $PIDFILE ]; then
+        if ps -p `cat $PIDFILE` | grep -q omsagent
+        then
```

Comment:
Please use the same style throughout for your if/then lines; most of our code uses the same line:
```
if [ check ]; then
...
fi
```

Reply:
Sure.